snippets/language-stata.cson

```diff
@@ -96,3 +96,40 @@
     'body':
       """<<dd_version: ${1:1}>>
       """
+
```

Comment:
You want to add `source.webdoc.stata` here so that these snippets don't come up within dyndoc files.

Reply:
ah yes, good catch. I added it in the new commit, but for some reason they still show up in all the stata languages.  Any ideas?